src/components/Modals/NewMessageModal.jsx

```diff
@@ -0,0 +1,210 @@
+
+  // Modifies message upon input
+  const handleChange = (e) => {
+    setMessage({
+      ...message,
+      [e.target.id]: e.target.value
+    });
+  };
+
+  // Handles submit (awaiting functionality for this)
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (!message.title) {
+      setError('Please enter a value for Message Title');
+    } else if (!message.recipientPodUrl) {
+      setError('Please enter a value for Recipient Pod URL');
+    } else if (!message.message) {
+      setError('Please enter a value for the Message');
+    } else {
+      try {
+        await sendMessageTTL(session, message, podUrl);
+
+        setMessage({
+          recipientPodUrl: '',
+          title: '',
+          message: ''
+        });
+        setError('');
+        setSuccess(`Message successfully sent to ${message.recipientPodUrl}`);
+        setSuccessTimeout(true);
+        setTimeout(() => {
+          setSuccessTimeout(false);
+        }, 10000);
+      } catch (err) {
+        // TODO: Make sure invalid username is the only possible error
+        setError(err.message);
+      } finally {
+        setOriginalMessage('');
+      }
+    }
+
+    // Re-sorts messages when new message is added to outboxList
+    const outboxMessages = await getMessageTTL(session, 'Outbox', outboxList, podUrl);
+    const sortedOutbox = outboxMessages;
+    sortedOutbox.sort((a, b) => b.uploadDate - a.uploadDate);
+    setOutboxList(sortedOutbox);
+  };
+
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  return (
+    <Dialog
+      open={showModal}
+      aria-labelledby="new-message-modal"
+      onClose={() => setShowModal(false)}
+    >
+      <Box
+        noValidate
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+          padding: '20px',
+          minWidth: '50%',
+          minHeight: '90%'
+        }}
+      >
+        <form onSubmit={(e) => handleSubmit(e)} autoComplete="off">
+          <Typography display="flex" justifyContent="center" variant="h5">
+            {oldMessage ? 'Reply To' : 'New Message'}
+          </Typography>
+          <TextField
+            margin="normal"
+            value={message.recipientPodUrl}
+            type="text"
+            name="recipientPodUrl"
+            id="recipientPodUrl"
+            onChange={(e) => handleChange(e)}
+            required
+            autoFocus
+            label="To"
+            fullWidth
+          />
+
+          {/* <label htmlFor="title">Subject*: </label> */}
+          <TextField
+            margin="normal"
+            value={message.title}
+            type="text"
+            name="title"
+            id="title"
+            onChange={(e) => handleChange(e)}
+            required
+            label="Subject"
+            inputProps={{ maxLength: '24' }}
+            fullWidth
+          />
+
+          <TextField
```

Comment:
I don't think old message is supposed to be an editable field. It's just supposed to display previous messages in the chain.

Reply:
Also, I would say that "Previous Message" shouldn't be in this at all, unless we are replying to a message.   

This `TextField` should have `sx={{ display: oldMessage ? 'block (or whatever)' : 'none' }}`